### DISCUSSION TITLE
Dark / inverted background for header or footer?

### DISCUSSION THREAD

**Discussion by mandrasch** [picocss/pico#534] [Help]
Hi,

nice work on v2! 👏 👏 

I wanted to implement a theme with a "dark" header / footer like

- https://demo.ghost.io/
- https://fershad.com/

What is the best / easiest way to do this? Can this be done via picos CSS variables?

(I'm fine with light mode styling only, no need to support dark mode at this point)

Happy for any hints! Thanks!

**Comment by dev-willis** [comment]
You can override the theme for any element and its children by setting `data-theme="[light/dark]"` on the element.

**Reply by mandrasch** [reply]
Ah cool, thanks very much for the hint! 👍
